packages/mobile-app/data/accounts/handlers.ts

```diff
@@ -0,0 +1,50 @@
+import { f } from "data-facade";
+import { z } from "zod";
+import { AccountsMethods } from "./types";
+
+const accounts = ["alice", "bob", "carol"];
+
```

Comment:
I am expecting that we have a handlers file per datasource? ie /accounts/mock.ts, /accounts/walletServer.ts, etc?